schema/me/conversation/message.js

```diff
@@ -77,10 +77,13 @@ export const MessageType = new GraphQLObjectType({
     },
 
     body: {
-      description: "Text which has been parsed/sanitized of quoted replies (among other things).",
+      description: "Unaltered text if possible, otherwise `body`: a parsed/sanitized version from Sendgrid.",
       type: GraphQLString,
-      resolve: ({ body }) => {
-        return body && body.split("\n\nAbout this collector")[0]
```

Comment:
This can go away now. In Emission, we'll replace this with `intitial_message` from the conversation, for the first message in the thread. We already are rendering that in a special way (artwork/show previews). I'll send that PR to you.